src/main/java/frc/robot/Drivetrain.java

```diff
@@ -26,11 +26,13 @@
   private final Translation2d m_backLeftLocation = new Translation2d(-0.381, 0.381);
   private final Translation2d m_backRightLocation = new Translation2d(-0.381, -0.381);
 
+  // Todo: these drive motor channel IDs need to be set
```

Comment:
We need to address these todos before deploying the code.

Please remove after it is done.